docs/articles/expensify-classic/expensify-billing/Receipt-Breakdown.md

```diff
@@ -1,56 +1,70 @@
 
-## High-Level Summary  
+- Shows the **total amount paid** across all workspaces under your billing ownership.
+- Displays the **price per member**, based on total active members.
+- Lists which members generated **billable activity** for the month.
 
-- Displays the **total amount paid** for all Expensify Workspaces under your billing ownership.  
-- Shows the **price per member**, calculated based on the total number of members across your workspaces.  
-- Provides a **member activity breakdown** to show which members generated billable activity.  
+---
 
----  
+# Save Money & Earn Cash Back
 
-## Reduce Your Bill & Get Paid to Use Expensify  
+Use the **Expensify Card** and an **annual subscription** to lower your costs and earn rewards.
 
-- Expensify Card users may qualify for **cash back** and **discounts** on their bill.  
-- This section shows **potential savings** if you use the Expensify Card for business expenses.  
-- Click **Get started** to connect with your account manager or Concierge for setup assistance.  
+This section of your receipt highlights:
+- **Estimated savings** from using the Expensify Card.
+- A button to **Get started**, which connects you with Concierge or your account manager for help setting up.
 
-**Note:** [Expensify Cards]([url](https://use.expensify.com/company-credit-card)) are currently available to companies with **US business bank accounts**.  
+**Note:** [Expensify Cards](https://use.expensify.com/company-credit-card) are only available to companies with a **US business bank account**.
 
----  
+---
 
-## Billing Breakdown  
+# Billing Breakdown
 
-Your receipt provides a **line-by-line breakdown** of charges and discounts:  
+This section outlines all charges and credits applied to your account:
 
-- **Inactive workspace members @ $0.00** – Users who did not generate billable activity.  
-- **Chat-only members @ $0.00** – Users who only used Expensify Chat. [Learn more](https://help.expensify.com/new-expensify/hubs/chat/).  
-- **Annual Control members @ $18.00** – Members covered under an annual **Control** plan subscription.  
-- **Pay-per-use Control members @ $36.00** – Additional **Control** plan members billed at a per-use rate.  
-- **Annual Collect members @ $10.00** – Members covered under an annual **Collect** plan subscription.  
-- **Pay-per-use Collect members @ $20.00** – Additional **Collect** plan members billed at a per-use rate.  
+- **Inactive workspace members @ $0.00** – Users who didn’t generate billable activity.
+- **Chat-only members @ $0.00** – Users who only used Expensify Chat.  
+  [Learn more](https://help.expensify.com/new-expensify/hubs/chat/)
+- **Annual Control members @ $18.00** – Members on an annual Control Plan.
+- **Pay-per-use Control members @ $36.00** – Extra Control members billed individually.
+- **Annual Collect members @ $10.00** – Members on an annual Collect Plan.
+- **Pay-per-use Collect members @ $5.00** – Intro rate for new users; conditions apply. 
```

Comment:
```suggestion
- **Collect members @ $5.00** – Intro rate for new users; conditions apply. 
```
NAB but technically the new line item looks like this:
![Screenshot 2025-04-15 at 2 28 42 PM](https://github.com/user-attachments/assets/9589fa82-48ad-44dd-84bf-d0e58de08491)